README.md

```diff
@@ -53,7 +53,7 @@ pip install git+https://github.com/huggingface/optimum-habana.git
 > ```bash
 > git clone https://github.com/huggingface/optimum-habana.git
 > cd optimum-habana
-> python setup.py install
+> pip install -e .
 > ```
```

Comment:
I would actually remove the whole comment from line 52 to line 57. `python setup.py install` doesn't work while `pip install` does.

Reply:
got it ..will do